examples/docker-compose/create.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	realis "github.com/paypal/gorealis"
+	"github.com/paypal/gorealis/gen-go/apache/aurora"
+)
+
+func main() {
+	r, err := realis.NewRealisClient(
+		realis.SchedulerUrl("localhost"),
+		realis.BasicAuth("aurora", "password"),
+		realis.ThriftJSON(),
+		realis.TimeoutMS(20000),
+		realis.Debug(),
+	)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	job := realis.NewJob().
+		Environment("prod").
+		Role("vagrant").
+		Name("sampleapp").
+		ExecutorName("docker-compose-executor").
+		ExecutorData("{}").
+		CPU(0.25).
+		RAM(256).
+		Disk(100).
+		IsService(true).
+		InstanceCount(1).
+		AddPorts(4).
+		AddLabel("fileName", "sampleapp/docker-compose.yml,sampleapp/docker-compose-healthcheck.yml").
+		AddURIs(true, false, "/dce-go/examples/sampleapp.tar.gz")
+
+	fmt.Printf("Creating job %v\n", job.JobKey())
+
+	resp, err := r.CreateJob(job)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	fmt.Println(resp.String())
+
+	monitor := &realis.Monitor{Client: r}
+	if resp.ResponseCode == aurora.ResponseCode_OK {
+		if ok, err := monitor.Instances(job.JobKey(), job.GetInstanceCount(), 5, 500); !ok || err != nil {
+			_, err := r.KillJob(job.JobKey())
+			if err != nil {
+				fmt.Println(err)
+				os.Exit(1)
+			}
+		}
+	}
+}
```

Comment:
create.go and kill.go is kind of duplicate with code in [client.go ](https://github.com/paypal/dce-go/blob/5fbd41d3982422c6c531989a1c3e60540fc62bf0/examples/client.go)  Do you think it make more sense to reuse or modify the exist one?